Sources/Analytics/Configuration.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Configuration.swift
+//  Alicerce
+//
+//  Created by Luís Portela on 26/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+extension Analytics {
+    
+    /// Anaylitics configuration.
+    /// Used to configure values in the `Analytics`
+    /// 💣 QueueQoS 👉 The qos to be used by the analytics queue
```

Comment:
Ok, at least that 😄 

Anyway, even though I am a fan of emoji 💪 I think that putting them like this on the documentation is probably going too far. But that's my opinion 😇 

Besides one of them being a bomb 😆, they won't be consistent with the rest of the documentation, so I would remove them